pandas/api/indexers/__init__.py

```diff
@@ -2,7 +2,7 @@
 Public API for Rolling Window Indexers.
 """
 
-from pandas.core.indexers import check_bool_array_indexer
+from pandas.core.indexers import check_array_indexer, check_bool_array_indexer
```

Comment:
Is there any value in keeping both?

Reply:
> Is there any value in keeping both?

No, I don't think so. Just need to clean-up if we are agree on the path forward.

Reply:
Yeah, that seems fine.